src/components/discover-sheet/DiscoverSearchContainer.js

```diff
@@ -63,7 +64,22 @@ export default forwardRef(function DiscoverSearchContainer(
     [setIsSearchModeEnabled, setShowSearch, setViewPagerSwipeEnabled]
   );
 
-  upperContext.onFabSearch.current = setIsInputFocused;
+  const onTapSearch = useCallback(() => {
+    if (isSearchModeEnabled) {
+      sectionListRef.current.scrollToLocation({
```

Comment:
.current?

Reply:
There's no need for that! This will be always present